### ISSUE TITLE
Clarification on Dataset Structure for Fine-Tuning

### ISSUE THREAD

**Issue by helia77** [bwittmann/vesselFM#2]
Hi,
I would like to fine-tune the model on my own dataset, but it's unclear how to structure the data within the dataset folder. In examples (eval_smile.yaml, eval_msd8.yaml, etc.), it seems that the data should be placed under an evaluation set directory, but there are no details on how the files should be organized inside.

Could you clarify:
- What is the expected folder structure for datasets like eval_smile?

- Should the data be split into train/, val/, and test/ subdirectories? (The finetune.py script uses a train folder.)

- What is the required format for the image and mask files (e.g., .npy, .nii, .png)?

Thank you.

**Comment by tobychenkaka** [comment]
Have you solved this problem? I also want to know how to name the data set used for fine-tuning and how to put it in the folder?

**Comment by bwittmann** [comment]
Hi @helia77 & @tobychenkaka,

Please note that the provided fine-tuning script is rather generic. Assuming that you would like to fine-tune vesselFM on in-house vascular data, **I strongly encourage you to plug vesselFM into a customized pipeline with augmentations and other parameters precisely matching the characteristics (and amount) of your data** to maximize performance.

*What is the expected folder structure for datasets like eval_smile? Should the data be split into train/, val/, and test/ subdirectories? What is the required format for the image and mask files (e.g., .npy, .nii, .png)?*: 

If you still decide to utilize our generic fine-tuning script, one needs a test, train, & val sub-folder. The structure would be like this:
```
eval_xxx/
└── train/
    └── sample_0/
        └── img.npy
        └── mask.npy
    └── sample_1/
        └── img.npy
        └── mask.npy
    └── sample_2/
        └── img.npy
        └── mask.npy
└── val/
    └── sample_0/
        └── img.npy
        └── mask.npy
└── test/
...
```

Hope this helps!
